File: Challenges/Java/RotateImage.java
final class RotateImage {

	int[][] rotateImage(int[][] a) {
	    int lh = a.length, lw = a[0].length;
	    int[][] out = new int[lh][lw];
	    
	    for(int i = 0, x = 0; i < lw; i++, x++) {
	        for(int j = lh - 1, y = 0; j >= 0; j--, y++) {
	            out[i][j] = a[y][x];
	        }   
	    }
	    
	    return out;
	}
}

File: Challenges/Javascript/rotateImage.js
function rotateImage(a) {
    let h = a.length, w = a[0].length, out = [];
    
    for(let i = 0, r = []; i < h; i++, r = []) {
        for(let j = w - 1; j >= 0; j--) r.push(a[j][i]);
        out.push(r);
    }
    
    return out;
}